core/kogitoservice/deployer_resources.go

```diff
@@ -62,72 +54,63 @@ func (s *serviceDeployer) createRequiredResources() (resources map[reflect.Type]
 		} else if !exists {
 			return resources, fmt.Errorf("propertiesConfigMap %s not found", s.instance.GetSpec().GetPropertiesConfigMap())
 		} else {
-			util.AppendToStringMap(getAppPropsFromConfigMap(propertiesConfigMap, exists), appProps)
+			util.AppendToStringMap(getAppPropsFromConfigMap(propertiesConfigMap), appProps)
 		}
 	}
 
-	_, configMap, err := appPropsConfigMapHandler.GetAppPropConfigMapContentHash(s.instance, appProps)
```

Comment:
Why is this block removed?

Reply:
Its redundent. 
https://github.com/vaibhavjainwiz/kogito-operator/blob/c6419ac54b5d2d1d75bd35cc8160f30432b5bc5d/core/kogitoservice/deployer_resources.go#L96